### ISSUE TITLE
EASE Jython needs to be moved from target to Dev menu

### ISSUE THREAD

**Issue by jayjaybillings** [eclipse/ice#259]
The EASE Jython target needs to be moved to the Developer menu. As discussed in #258, it is only approved for a Works-With CQ and we cannot distribute it in our binary.

**Comment by jarrah42** [comment]
I think we did this for a tutorial, but it should be simple to move it back. Any takers or should I do it?

**Comment by jayjaybillings** [comment]
@jarrah42 It's all you.

**Comment by jarrah42** [comment]
I thought that might be the case...

**Comment by jarrah42** [comment]
Depends on #309

**Comment by jayjaybillings** [comment]
How's this going? I got an email from the IP team about it yesterday.

**Comment by jayjaybillings** [comment]
Will someone take a moment to summarize the status of this and #309 for me please? I need to get 2.2.1 out the door soon and both of these bugs are in the way. I think six days ago @amccaskey said he was going to push Greg's dev menu changes. Where does that leave this bug?

**Comment by jarrah42** [comment]
Now that the changes are merged, a developer menu to install EASE needs to be added. There's no harm in shipping 2.2.1 without this, however.

Is there a release date for 2.2.1? I haven't seen a plan anywhere.

**Comment by jayjaybillings** [comment]
The release date was December 5th for 2.2.1. All of the plans were submitted to the PMC and announced to the dev list about a week before that. I set the date in an email to the team before we actually released 2.2.0. There's also a GitHub milestone on it.

We can't ship 2.2.1 without this because the IP team has asked us to fix it first. Release reviews are not required for maintenance releases, but I put one in anyway to teach Robert and Hilda how to do it. Once I submitted that the IP noticed the error and requested it be fixed before 2.2.1 is released.

Will you add the developer menu changes and remove Jython from the target please or should I ask someone else to tackle it?

**Comment by jarrah42** [comment]
A note about missing the release date would probably have cleared up any confusion. I thought there must have been a new release date since then.

It's been fixed from the IP perspective for some time as the offending plugin was removed on 12/8. The only issue is how important it is to have the EASE developer menu in this release. I can probably do it by early next week, but if you need it sooner then someone else should do it.

**Comment by jayjaybillings** [comment]
A note probably would have, but we discussed it during several status meetings too. Don't sweat it though - this normally happens when holidays get in the way.

Okay, since the offending plugin was removed I'll move forward on the maintenance release. To be honest, I'm not even sure that we need to put this in the developer menu if the EASE Marketplace is still available under Help. It is just prevents interface overload for someone wanting to get Python scripting going to click it directly in the developer menu.
